contest/templatetags/contest_extras.py

```diff
@@ -32,3 +38,72 @@ def has_auth(user,contest_id):
     return user_info.has_c_ownership(user,contest)
 
```

Comment:
Please unify your coding style: add a blank line before register.filter... or not.